scripts/qa_crowbarsetup.sh

```diff
@@ -2908,6 +2908,18 @@ function onneutron_wait_for_neutron()
     fi
 }
 
+function get_nova_multi_controller_node()
```

Comment:
@aspiers Done in https://github.com/toabctl/automation/commit/4bf09bd8231e92f7f0c249a4473c45ff344c9e58 .